src/main/java/com/savvato/tribeapp/repositories/PrepositionRepository.java

```diff
@@ -1,13 +1,18 @@
 package com.savvato.tribeapp.repositories;
 
 import com.savvato.tribeapp.entities.Adverb;
+import com.savvato.tribeapp.entities.Preposition;
 import com.savvato.tribeapp.entities.Verb;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
 import java.util.Optional;
 
 @Repository
-public interface PrepositionRepository extends CrudRepository<Adverb, Long> {
-    Optional<Verb> findByWord(String word);
+public interface PrepositionRepository extends CrudRepository<Preposition, Long> {
+    Optional<Preposition> findByWord(String word);
+
+    @Query(nativeQuery = true, value = "select word from preposition where id = ?")
+    Optional<String> findPrepositionById(Long id);
```

Comment:
should be findById(id).. no custom query..

Reply:
Will fix according to what we decide in Adverb conversation above.